### ISSUE TITLE
Data Agumentation

### ISSUE THREAD

**Issue by haoranD** [pxq0312/SFANet-crowd-counting#1]
Hi, thanks for your excellent work! I just ran your code directly without any changes(except the path to data).

But I only got MAE7.58 on ShanghaiTech Part B. Can you please give me some suggestions? Any data augmentation plz? I really want to reproduce the 6.38 MAE. Thanks for your help.

Thank you and have a good day. @pxq0312

**Comment by jiezishu737** [comment]
> Hi, thanks for your excellent work! I just ran your code directly without any changes(except the path to data).
> 
> But I only got MAE7.58 on ShanghaiTech Part B. Can you please give me some suggestions? Any data augmentation plz? I really want to reproduce the 6.38 MAE. Thanks for your help.
> 
> Thank you and have a good day. @pxq0312

The author has two projects. In the last project, i had the similar question. The author gave the answer that they put the whole training dataset as the traing set and put the testing dataset as the validation set. So the results is just a training result , not a testing result

**Comment by daimuuc** [comment]
> > Hi, thanks for your excellent work! I just ran your code directly without any changes(except the path to data).
> > But I only got MAE7.58 on ShanghaiTech Part B. Can you please give me some suggestions? Any data augmentation plz? I really want to reproduce the 6.38 MAE. Thanks for your help.
> > Thank you and have a good day. @pxq0312
> 
> The author has two projects. In the last project, i had the similar question. The author gave the answer that they put the whole training dataset as the traing set and put the testing dataset as the validation set. So the results is just a training result , not a testing result

Hi, my result is 75(MAE) on the SHA dataset. I found that the code does not have a learning rate adjustment strategy. Is this why?thanks

**Comment by lucky437** [comment]
Hello, thanks for your excellent work.I have  some questions I want to ask you.How did you get the visualization results in the paper, there is no code, hope to get a reply, thank you very much.

**Comment by hiramustafa77** [comment]
> Hello, thanks for your excellent work.I have some questions I want to ask you.How did you get the visualization results in the paper, there is no code, hope to get a reply, thank you very much.

Hi did you find out the solution to your question
